grunt/replace.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * (c) ECOLE POLYTECHNIQUE FEDERALE DE LAUSANNE, Switzerland, VPSI, 2019.
+ * See the LICENSE file for more details.
+ */
+
+'use strict';
+
+module.exports = {
+  release: {
+    options: {
+      patterns: [{
+        match: /<FONTS_PATH>/g,
+        replacement: 'https://web2018.epfl.ch/2.0.0/fonts/'
```

Comment:
remove last slash '/' ? We have thw '/' in url generated.
-- ex url: https://web2018.epfl.ch/2.0.0/fonts//SuisseIntl-RegularItalic-WebS.svg#svgFontName

Reply:
Nice catch.